input/fsh/Alias.fsh

```diff
@@ -51,3 +51,8 @@ Alias: $disease-codes = http://fhir.ncpi-project-forge.io/ValueSet/disease-codes
 Alias: $phenotype-codes = http://fhir.ncpi-project-forge.io/ValueSet/phenotype-codes
 Alias: $loinc-answer-codes = http://fhir.ncpi-project-forge.io/ValueSet/loinc-answers
 Alias: $condition-interitanc-codes = http://fhir.ncpi-project-forge.io/ValueSet/condition-inheritance-vs
+
+// NCPI Observations
+Alias: $ncpi = https://www.ncbi.nlm.nih.gov/fhir
```

Comment:
I'm not sure this is actually the correct URL for ncpi. You may have gotten that from my examples, which were put together rather hastily. I only noticed I had grabbed nc-b-i after I had sent those files off.